code/__DEFINES/_init.dm

```diff
@@ -4,7 +4,7 @@
 	// This exists so that world.Profile() is THE FIRST PROC TO RUN in the init sequence.
 	// This allows us to get the real details of everything lagging at server start.
 	world.Profile(PROFILE_START)
-	#if defined(ENABLE_BYOND_TRACY) && (DM_BUILD == 1589)
+	#if defined(ENABLE_BYOND_TRACY)
```

Comment:
```suggestion
	#if defined(ENABLE_BYOND_TRACY) && (DM_BUILD == 1623)
```

?

Reply:
Nah it works with many versions, I just had it locked at this at the time to prevent slipups.